### DISCUSSION TITLE
My question was deleted and I am prohibited from asking again.

### DISCUSSION THREAD

**Discussion by will-bvs** [community/community#129048] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I raised a product issue on Github, my issue was deleted and my account was blocked from creating issues.

I am confused about this.

author: @Pszz
discussions url: https://github.com/orgs/community/discussions/128833
^

I just made suggestions for product improvements.

There is no need to force users to change their passwords frequently. 

My password has changed from a combination I am familiar with to an unknown combination, which makes it impossible for me to remember it.

You can alert users to the risks, 
but forcing them to change their passwords or forcing them to enable 2FA is a terrible product.

And I have already enabled 2FA, and I believe it is safe enough.

**Comment by will-bvs** [comment]
If GitHub doesn't allow users to ask questions freely, then I can remain silent.

**Comment by asdacosta** [comment]
Review GitHub’s guidelines to understand why this might have happened. Contact GitHub support for clarification.
